### PR TITLE
Fix js errors

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fix JS error (you can see it on the iterator doc page) and the invalid theme switch.

r? @QuietMisdreavus

### CODE REVIEW

File: src/librustdoc/html/static/storage.js [hunk 1]
```diff
@@ -101,6 +101,7 @@ function getCurrentValue(name) {
 }
 
 function switchTheme(styleElem, mainStyleElem, newTheme) {
+    console.log(newTheme);
```
Comment (Eijebong) [inline src/librustdoc/html/static/storage.js]: Hum :)

Comment (GuillaumeGomez) [reply src/librustdoc/html/static/storage.js]: Woups!

**Review by QuietMisdreavus — APPROVED** [review]
Confirmed this fixes the issues in personal testing. (The Iterator page is still really laggy though `>_>`)

**Pull request merged**